src/gqlRequest.ts

```diff
@@ -0,0 +1,120 @@
+import { ClientError, GraphQLClient } from 'graphql-request';
+import { GraphQLError } from 'graphql-request/dist/types';
+
+import {
+  getSdk as getAuthSdk,
+  Sdk as AuthSdk,
+} from 'src/types/generated/graphql.auth.types';
+import {
+  getSdk as getZonosCustomerGraphSdk,
+  Sdk as ZonosCustomerGraphSdk,
+} from 'src/types/generated/graphql.zonos-customer-graph.types';
+
+type Schema = 'zonos-customer-graph' | 'auth';
```

Comment:
What is the reason for the `zonos` prefix on one and not the other?

Reply:
😆 Good catch. Thanks